packages/flutter/lib/src/widgets/restoration_properties.dart

```diff
@@ -125,7 +125,50 @@ abstract class RestorableValue<T> extends RestorableProperty<T> {
 
   @mustCallSuper
   @override
-  void initWithValue(T value) {
+  void initWithValue(T? value) {
```

Comment:
Same question as above, if this needs to be nullable, wouldn't you just specify T as something nullable?